### PR TITLE
Introduce PoseGraphInterface::ToProto()

### PR DESCRIPTION
**Pull request by cschuet**

### CODE REVIEW

File: cartographer/mapping/pose_graph.h [hunk 1]
```diff
@@ -110,7 +110,7 @@ class PoseGraph : public PoseGraphInterface {
   virtual SubmapData GetSubmapData(const SubmapId& submap_id) = 0;
 
   // Serializes the constraints and trajectories.
```
Comment (gaschler) [inline cartographer/mapping/pose_graph.h]: remove comment

**Comment by cschuet** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user cschuet. Merge queue now has a length of 2.

**Comment by zjwoody** [comment]
LGTM. Thanks!

**Pull request merged**